client/blocks/taxonomy-manager/style.scss

```diff
@@ -36,63 +36,70 @@
 
 .taxonomy-manager__list-item-card {
 	cursor: pointer;
+	padding: 0 !important;
 }
 
-.taxonomy-manager__action-wrapper {
-	cursor: pointer;
-	font-size: 24px;
-	padding: 10px 28px;
-	position: absolute;
-		right: 0;
-		top: 0;
-}
-
-.taxonomy-manager__list-item-toggle {
-	color: $gray;
-	transition: all 0.15s cubic-bezier(0.175, 0.885, 0.32, 1.275);
-
-	&.is-active {
-		color: $blue-medium;
-		transform: rotate( 90deg );
-	}
-}
-
-.taxonomy-manager__label {
-	margin-left: 30px;
-	line-height: 18px;
+.taxonomy-manager__item {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
 
-	.taxonomy-manager__list-item.is-placeholder & {
-		color: transparent;
-		background-color: lighten( $gray, 30% );
-		animation: loading-fade 1.6s ease-in-out infinite;
+	.taxonomy-manager__icon {
+		margin: 15px;
+		color: $gray;
+		.gridicon {
+			display: block;
+		}
 	}
-
-	.taxonomy-manager__default-label {
-		text-transform: uppercase;
+	&.is-default .taxonomy-manager__icon {
 		color: $blue-medium;
-		font-size: 12px;
-		margin-left: 10px;
 	}
-}
 
-.taxonomy-manager__item {
-	.taxonomy-manager__icon {
-		margin: 15px 20px 0;
-		position: absolute;
-			left: 0px;
-			top: 0;
-		color: $gray;
+	.taxonomy-manager__label {
+		line-height: 18px;
+		white-space: nowrap;
+		overflow: hidden;
+		flex-grow: 1;
+		position: relative;
+
+		&::after {
+			@include long-content-fade();
+		}
+
+		.taxonomy-manager__list-item.is-placeholder & {
+			color: transparent;
+			background-color: lighten( $gray, 30% );
+			animation: loading-fade 1.6s ease-in-out infinite;
+		}
+
+		.taxonomy-manager__default-label {
+			text-transform: uppercase;
+			color: $blue-medium;
+			font-size: 12px;
+			margin-left: 10px;
+		}
 	}
 
 	.count {
-		margin: 18px 28px;
-		position: absolute;
-			right: 48px;
-			top: 0;
+		margin: 18px 2px;
 	}
 
-	&.is-default .taxonomy-manager__icon {
-		color: $blue-medium;
+	.taxonomy-manager__action-wrapper {
+		cursor: pointer;
+		font-size: 24px;
+		padding: 15px 28px;
```

Comment:
These numbers are not arbitrary, since they can have impact on the height of each term row. But I guess with the new "Flex" technique, I can probably drop the majority of them .